Module_1/getelement.html

```diff
@@ -0,0 +1,42 @@
+<!-- Script tag at start of head -->
+<!DOCTYPE html>
+<html>
+<head>
+    <script>
+        document.getElementById("test").innerHTML = "Goodbye World";
+    </script>
+</head>
+<body>
+<h1 id="test">Hello World!</h1>
+</body>
+</html>
+
+
+<!-- Script tag at start of body -->
+<!DOCTYPE html>
+<html>
+<head></head>
+<body>
+    <script>
+        document.getElementById("test").innerHTML = "Goodbye World";
+    </script>
+    <h1 id="test">Hello World!</h1>
+</body>
+</html>
+
+<!-- Script tag at end of body -->
+<!DOCTYPE html>
+<html>
+<head></head>
+<body>
+    <h1 id="test">Hello World!</h1>
+    <script>
+        document.getElementById("test").innerHTML = "Goodbye World";
+    </script>
+</body>
+</html>
+
+
```

Comment:
At end of the code, can you also add comments stating what you observed. For example what was the output of this code. Did the hello world changed to Goodbye World etc